### PR TITLE
#23 explained main goal of the library in details

### PR DESCRIPTION
**Pull request by dmitrykrivaltsevich**
part of #23

### CODE REVIEW

**Comment by wokuyama** [comment]
👍

**Review by wokuyama — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -16,26 +16,51 @@
...
 - Declarative security configuration via configuration file.
 - Minimalistic. The library does only one thing, but does it good.
 - Developers friendly. Just plug it in and you are done.
-- Non-blocking from top to bottom.
+- Easy to use in tests with `AlwaysPassAuthProvider` provider.
 - Opinionated design choice: the library relies on `Play`'s toolbox like `WS`-client from the beginning because it is designed to work _exclusively_ within Play-applications.
 
 We mainly decided to release this library because we saw the needs of OAuth2 protection in almost every microservice we built and because we are not happy to couple our code with any forms of custom security actions or annotations.
 
+## Known alternatives
```
Comment (raychenon) [inline README.md:60]: You added some titles. The table of content changed :)
The new one
```
**[Table of Contents](http://tableofcontent.eu)**
<!-- Table of contents generated generated by http://tableofcontent.eu -->
- [Play-security library](#play-security-library)
  - [Core Technical Concepts](#core-technical-concepts)
  - [Known alternatives](#known-alternatives)
  - [Getting Started](#getting-started)
  - [Contributing](#contributing)
  - [Contact](#contact)
  - [License](#license)
```

Comment (dmitrykrivaltsevich) [reply README.md:60]: thanks, @raychenon . Please review again :)

**Comment by raychenon** [comment]
👍

**Pull request merged**